_posts/2021-03-31-hhvm-4.103.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: "HHVM 4.103"
+layout: post
+author: fred
+category: blog
+---
+
+HHVM 4.103 is released! This release marks the end of support for 4.96;
```

Comment:
```suggestion
HHVM 4.103 is released! This release marks the end of support for 4.97;
```

Reply:
Hmm, how come? 4.97 is 6 weeks old, wasn't an LTS, and 4.102 also said 4.97

Reply:
4.102 announcement says "end of support for 4.96", that's why I thought this should be 97

There should be 6 supported releases (+any older LTS), so we have 98, 99, 100, 101, 102, 103 now

Reply:
I misread, I thought you were suggesting the opposite to this change :)